yang/frr-staticd.yang

```diff
@@ -0,0 +1,150 @@
+module frr-staticd {
+  yang-version "1.1";
+  namespace "http://frrouting.org/yang/staticd";
+
+  prefix frr-staticd;
+
+  import ietf-routing-types {
+    prefix "rt-types";
+  }
+
+  import ietf-routing {
+    prefix "rt";
+  }
+
+  import ietf-ipv4-unicast-routing {
+    prefix "v4ur";
+  }
+
+  import ietf-ipv6-unicast-routing {
+    prefix "v6ur";
+  }
+
+  import frr-vrf {
+    prefix frr-vrf;
```

Comment:
If I'm not mistaken this prefix statement isn't required, since it's the same as the module name.